DSharpPlus.Commands/Processors/BaseCommandProcessor/BaseCommandProcessor.cs

```diff
@@ -200,6 +200,10 @@ public virtual ValueTask ConfigureAsync(CommandsExtension extension)
         return default;
     }
 
+    /// <inheritdoc />
+    public static Type ContextType { get; }
+
+

```

Comment:
Why is this in the middle of methods :ioa:

Also potentially consider making it a computer property of `typeof(TCommandContext)`? Can't see when it would ever be the case that the context type would differ from what's specified as the generic

Reply:
good idea